cypress/e2e/focus-editor-placeholder.cy.js

```diff
@@ -0,0 +1,10 @@
+describe("Clicking on editor placeholder must not lose the focus on editor", () => {
+    beforeEach(() => {
+        cy.visit("/");
+    });
+
+    it("Clicking on placeholder text should not lose focus on editor", () => {
```

Comment:
I'd reword `it("Clicking on placeholder text should not lose focus on editor", () => {` to `it("should retain focus on the editor when clicking on the placeholder text", () => {`.

Reply:
I guess when we click as a user, we don't really do the `cy.get('[data-test="editor-placeholder-text"]').click();` action? One alternative I guess is to fetch the coordinates of the placeholder text we want to click on, and then click on it based on these coordinates,
```
cy.get(el).then($el => {
                let coords = $el[0].getBoundingClientRect();
            })
```
see also https://docs.cypress.io/api/commands/click#Syntax

Reply:
Oh and I'd start this test by doing `cy.get('[data-test="contact"]').click();` since I think it makes sense to remove the editor auto-focus on page load behavior, and simply test this feature.